pkg/handlers/session.go

```diff
@@ -85,6 +85,20 @@ func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.R
 		return nil, err
 	}
 
+	passwordUpdatedAt, err := kotsStore.GetPasswordUpdatedAt()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get password updated at")
+	}
+	if passwordUpdatedAt.After(sess.IssuedAt) {
+		if err := kotsStore.DeleteSession(sess.ID); err != nil {
+			logger.Error(errors.Wrapf(err, "password was updated after session created. failed to delete invalid session %s", sess.ID))
+		}
+		err := errors.New("password changed, please login again")
+		response := ErrorResponse{Error: err.Error()}
+		JSON(w, http.StatusUnauthorized, response)
+		return nil, err
+	}
+
```

Comment:
isn't this redundant if the sessions are being deleted?

Reply:
if delete sessions fail, this would be a good fallback option 
 additionally, addresses @trelore [comment](https://github.com/replicatedhq/kots/pull/2739#discussion_r856173161)